l2rpn_baselines/Template/__init__.py

```diff
@@ -18,7 +18,7 @@
 - `evaluate` [**mandatory**] contains the script to evaluate the performance of this baseline. It must
   follow the directive in "evaluate.py"
 - `train` [**optional**] contains the script to train your baseline. If provided, it must follow
-  the directives given in "train.py"
+  the directives given in "SAC.py"
```

Comment:
Again, roll back to train.py

Reply:
Ok i made a warning to not use this one. We'll remove it later, to let people some time to swtich